Misc/NEWS.d/next/Library/2018-02-21-04-39-49.bpo-32888.G5i41Q.rst

```diff
@@ -0,0 +1,2 @@
+Improve wording of error messages from ast.literal_eval, distinguishing
+valid-but-unacceptable nodes from those that are actually malformed.
```

Comment:
Please add "Patch by..."

Reply:
Should that be done by amending the current commit, or adding an additional commit?

Reply:
It is better for review to add a new commit. In any case all commits are squashed before merging into the repository.